### DISCUSSION TITLE
Where is the Multiple Images tab in ControlNet?

### DISCUSSION THREAD

**Discussion by sohamaa** [lllyasviel/stable-diffusion-webui-forge#344] [Q&A] [answered]
Sorry if it's a noob one, but why I can't find this section in the Photomaker tab as you showed? 
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/119129683/c555ea17-fc30-475f-aa5f-6578c74c8d1b)

for me it's just a batch upload, are they the same?
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/119129683/34e16b0e-e35c-42ea-a086-b206efc4be44)

**Comment by huchenlei** [comment, marked as answer]
See https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/264

**Reply by strawberrymelonpanda** [reply]
To be fair, they raise a very valid point: The "Multiple Images" tab is in the screenshots in the README.

**Reply by sohamaa** [reply]
Thank you so much! Now it's back and also it works well!
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/119129683/83fc70c8-0906-449c-ae95-e9530db0f5e3)

For those who like me, I downloaded these 5 files from here:
https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/264/files

Like this: (Get download file by clicking (view file)
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/119129683/0256ae4f-4f46-43f5-99e6-62c126c514fd)

and put it in the right directory. Then close the terminal and start again (Restart UI won't work) 
Thanks to @huchenlei

**Comment by tealytee** [comment]
I get this errror, how do i fix it
*** Error loading script: controlnet.py
    Traceback (most recent call last):
      File "F:\SD FORGE\sd forge\webui\modules\scripts.py", line 525, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
      File "F:\SD FORGE\sd forge\webui\modules\script_loading.py", line 13, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "F:\SD FORGE\sd forge\webui\extensions-builtin\sd_forge_controlnet\scripts\controlnet.py", line 15, in <module>
        from lib_controlnet.utils import (
      File "F:\SD FORGE\sd forge\webui\extensions-builtin\sd_forge_controlnet\lib_controlnet\utils.py", line 9, in <module>
        from modules_forge.forge_util import HWC3
    ModuleNotFoundError: No module named 'modules_forge.forge_util'
